### PR TITLE
Webpack Running/Created Simple Test

### PR DESCRIPTION
**Pull request by tejones112**

### CODE REVIEW

**Comment by lennee** [comment]
Looks Good! Merging to master

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          presets: ['@babel/react', '@babel/env']
+        }
+        // use: {
+        //   loader: "babel-loader",
+        //   options: {
+        //     presets: ['@babel/react', '@babel/env']
+        //   }
+        // }
+      },
+    ]
```
Comment (ryancmoniz) [inline webpack.config.js:27]: Make sure you remove comments that are no longer necessary to your code.

File: server/index.js [hunk 2]
```diff
@@ -7,11 +7,12 @@ const app = express();
...
 
 //REMEMBER TO FIX FIND ONE BELOW!!!!!!!!!!!!
-app.get('/videoData', (req, res) => {
-  mongoModel.GameModel.findOne({id: 1})
+app.get('/api/mediaData/:id', (req, res) => {
+  let id = req.params.id
+  mongoModel.GameModel.findOne({id})
     .then((data) => {
       res.json(data);
     })
```
Comment (ryancmoniz) [inline server/index.js:18]: Is there something better we can do when an error occurs besides logging it to the console? As a user, I want to know why my get request to id 90000001 didn't work.

File: database/seed.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ function generateData() {
   }
   return games;
 }
-var test = generateData();
+var seededData = generateData();
```
Comment (ryancmoniz) [inline database/seed.js]: I like the name change to a variable that makes more sense as someone new to your codebase. Are you using a linter yet? I am seeing a lot of Let and Const which is good. Why is this line using Var?

**Pull request merged**